dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditUtil.java

```diff
@@ -95,25 +95,26 @@ public Contentlet findContentletByIdentifier ( String identifier ) throws DotSec
 
         //Getting the default language
         Language defaultLanguage = APILocator.getLanguageAPI().getDefaultLanguage();
+        Contentlet foundContentlet = null;
 
         /*
         We may added a contentlet that exist only in a NON default language so
         assuming the content will always exist in the default language is wrong.
          */
         List<Contentlet> allLanguages = APILocator.getContentletAPI().search( "+identifier:" + identifier, 0, 0, "moddate", user, false );
-
-        /*
-        For display purposes we are trying to return the contentlet with the default language, if
-        nothing found for the default language just return the first one.
-         */
-        Contentlet foundContentlet = null;
-        for ( Contentlet contentlet : allLanguages ) {
-            if ( contentlet.getLanguageId() == defaultLanguage.getId() ) {
-                foundContentlet = contentlet;
+        if(allLanguages.size()>0) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)

Reply:
We have on UtilsMethod a is set for collections